packages/tests/src/components/bd-checkbox.ts

```diff
@@ -0,0 +1,16 @@
+import BigDesignElement from './bd-element';
+
+// TODO: Make more typescripty
+export default class BigDesignCheckbox extends BigDesignElement {
+  toggle = (): void => {
```

Comment:
I would suggest to not manually annotate function return types unless you have a  compelling reason for it. Just let the compiler do its thing.

Also if you don't specifically need a bound function (arrow syntax) you can just do 

```
toggle() {
... 
}
```

Reply:
I thought in the other PR you were asking to use the bound function syntax... https://github.com/bigcommerce/big-design/pull/289#discussion_r355610086

Reply:
or is it the difference between an instance function and a class function?

Reply:
Yep sorry must've been more clear. That is precisely the difference here

What I meant in the previous comment is doing `foo(() => { ... })` instead of `foo(function() { ... })`, but for class instance methods `foo() { .. }` is the standard way

Reply:
✅